README.md

```diff
@@ -3,15 +3,15 @@ RxRiddles
 
 This repository contains some small for RxJava to help you learn and master RxJava. There is an [accompanying medium article](https://medium.com/@vanniktech/riddling-your-way-to-master-rxjava-145d5de99b55) that gives some more information.
 
-Each riddle is in a single file with an accompanied unit test that will check your implementation. Next to the unit test I have also put up my solution. Note that there are multiple ways to achieve and solve the riddles but usually there's a dedicated operator or function that I want to show you.
+Each riddle is in a single file with an accompanying unit test that will check your implementation. Next to the unit test, I have also put up my solution. Note that there are multiple ways to achieve and solve the riddles but usually there's a dedicated operator or function that I want to show you.
 
 The riddles are not sorted in any real preference and I plan to keep it that way. Just start with whichever one you prefer. Riddles with numbers lower than 100 can be solved with a single operator while every other riddle requires multiple operators.
 
 ### Contributing
 
-I'm very open to have some more riddles. Especially some complex common use cases. In case you want to contribute create an issue and let's talk.
+I'm very open to having some more riddles. Especially some complex common use cases. In case you want to contribute create an issue and let's talk.
```

Comment:
This should be "of having" then.

Reply:
Actually, I ran "I'm very open *of* having some more riddles." through Grammarly Chrome plugin and was notified that it should be "to" not "of". I can just revert my change to this sentence if you like.

Reply:
Alright then that works.